hackathon/templates/hackathon/hackathon_events.html

```diff
@@ -0,0 +1,30 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2>{{ hackathon.display_name }} - Webinars</h2>
+  {% if events %}
+    <div class="card">
+      <div class="card-body">
+        <ul class="list-group">
+          {% for event in events %}
+            <li class="list-group-item d-flex justify-content-between align-items-center">
+              <div>
+                <h5> {{ hackathon.display_name }} {{ event.title }}</h5>
+                <p>{{ event.start }}</p>
+                <p><a href="{{ event.webinar_link }}" target="_blank">Webinar Link </a></p>
+                <p>{{ event.body }}</p>
```

Comment:
I would maybe add a Jinja filter (e.g. `|safe`) in the tag to actually display the html and add a label like "Calendar Pop-up Body Preview" or similar. Just a thought though, no blocker from merging